modules/enterprise/agent/src/main/java/org/rhq/enterprise/agent/AgentConfigurationConstants.java

```diff
@@ -162,6 +162,16 @@
     String AGENT_UPDATE_DOWNLOAD_URL = PROPERTY_NAME_PREFIX + "agent-update.download-url";
 
     /**
+     * The amount of milliseconds the agent will wait before force closing after an update.
+     */
+    String AGENT_UPDATE_EXIT_TIMEOUT_MSECS = PROPERTY_NAME_PREFIX + "agent-update-exit-timeout-msecs";
+
```

Comment:
name of the field is fine, but the value should be "agent-update.exit-timeout-msecs" - notice the "." after agent-update. The way these are define is we have the name of a group of properties followed by "." followed by the name of the property. Notice we have several agent-update properties - you are adding another:

agent-update.enabled
agent-update.version-url
agent-update.download-url
agent-update.exit-timeout-msecs <-- you want a "." after agent-update, like all the rest in the group.